ibm/mas_devops/roles/dro/README.md

```diff
@@ -59,6 +52,34 @@ Local directory to save the generated BasCfg resource definition.  This can be u
 - Environment Variable: `MAS_CONFIG_DIR`
 - Default Value: None
 
+### dro_existing
```

Comment:
What's the purpose of this?  In this scenario I would expect teh action to be that the user sets `uds_action=none` so that DRO is not installed, and pass in the required BASCfg with the addtional configurations mechinism supported in suite_config, so that this is consistent with how you would configure an external mongo, db2, or kafka instance.  We don't need to create another code path that needs to be maintained/tested for achieving the same thing.

Reply:
CLI does not support BYO UDS, DRO, SLS, or Mongo today.  These installations are only supporting by using the ansible roles/building your own playbook.

Reply:
If we want to add support for them refer to the pattern established for the JDBCCfg management, whereby the user is promoted to generate the configuration and it's added to the additional configs metadata.

Reply:
@durera yea the BYO DRO is support on ansible roles. CLI does not have it. do u expect dro_existing to be removed from the ansible role as well?

Reply:
looks like @yuvraj-vansure already removed it.